crates/common/src/commit/client.rs

```diff
@@ -60,16 +65,13 @@ impl SignerClient {
             });
         }
 
-        let parsed_response: GetPubkeysResponse = serde_json::from_slice(&response_bytes)?;
+        let parsed_response = serde_json::from_slice(&response_bytes)?;
 
         Ok(parsed_response)
```

Comment:
```rust
pub async fn get_pubkeys(&self) -> Result<GetPubkeysResponse, SignerClientError> {
        let res = self.client.get(&format!("{}{}", self.url, GET_PUBKEYS_PATH)).send().await?;

        if !res.status().is_success() {
            return Err(SignerClientError::FailedRequest {
                status: res.status().as_u16(),
                error_msg: String::from_utf8_lossy(&res.bytes().await?).into_owned(),
            });
        }

        Ok(serde_json::from_slice(&res.bytes().await?)?)
    }
```

Reply:
Applied in [4e652d9](https://github.com/Commit-Boost/commit-boost-client/pull/87/commits/4e652d97d33ce7dc3051db072a0f0298878ee315)